PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
-
-1. {DEV_3} & {DEV_1}
-2. {DEV_4} & {DEV_2}
-
-#### Week 7, {DATES_WEEK_7}
-
-1. {DEV_1} & {DEV_2}
-2. {DEV_3} & {DEV_4}
-
-#### Week 8, {DATES_WEEK_8}
-
-1. {DEV_2} & {DEV_3}
-2. {DEV_1} & {DEV_4}
+### Week 1, {DATES_WEEK_1}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                                    |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| {DEV_1} & {DEV_2}     | [2. As a user, I want to read all of my shopping lists and the items in them]({REPO_URL}/issues/2)                                          |
+| {DEV_3} & {DEV_4}     | [3. As a user, I want to be able to navigate to all the pages in the application by clicking the links in the nav bar]({REPO_URL}/issues/3) |
+
+| 📖 Learning modules                                                                                                                                                                 | Description |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| [Git Presentation](https://youtu.be/zeC5iwsHllk) ([slides](https://docs.google.com/presentation/d/1onRsF-5NQbky8BOac5cGsJunjBisvauQPEC4hOWPaGI/edit#slide=id.p))                    |
+| [Code Review Presentation](https://youtu.be/QcItLjFJ_Us) ([slides](https://docs.google.com/presentation/d/1GP_X91rd3W50GN6FjcLfAge8SD0O45qe7qdtHPO_D2I/edit?pli=1#slide=id.p))      |
+| [Communication for Devs Presentation](https://youtu.be/LVjG3qxtQX0) ([slides](https://docs.google.com/presentation/d/14kJTPHCVHsEmzqOLTqKCgL3QrwR7_BnIpN9oGJraT4c/edit#slide=id.p)) |
+| [Pair Programming Presentation](https://youtu.be/yFVJ2gph_Uc) (slides)                                                                                                              |
+| Agile Presentation (slides only)                                                                                                                                                    |
+
+| 👋 Sync meetings | Agenda                                                                                                                                                                                                                                   |
+| ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Weekly Sync      | - Mentors pair live through issue #1 so Collabies can get a demostration of the full development cycle at Collab Lab and we can ensure everyone has Firebase auth for their app <br> - Collabies will be presented with issues #2 and #3 |
```

Comment:
The language around issues changes a bit from week to week:
- "Collabies will be presented with issues #2 and #3"
- "This week's issue"
- "Next week's issues"

Is that intentional?

Reply:
No I was just lazy and wanted to get the PR up. :sweat-smile: I'll update today! :heart: